README.md

```diff
@@ -6,15 +6,11 @@
          alt="">
 </a>
 
-Effects Tab is a [custom tab][uc-custom-tabs] for [Uploadcare Widget][uc-widget]
+Effects Tab is a [custom tab][uc-docs-widget-custom-tabs] for [Uploadcare Widget][uc-docs-widget]
 that replaces Preview Tab and adds image-editing capabilities to the widget.
```

Comment:
Effects Tab is an Uploadcare Widget addon that allows for in-browser image editing on desktop and mobile. Technically, it's a custom widget tab that replaces Preview Tab.